queue_and_stack/dll_stack.py

```diff
@@ -6,13 +6,18 @@ class Stack:
   def __init__(self):
     self.size = 0
     # Why is our DLL a good choice to store our elements?
-    self.storage = 
+    self.storage = []
```

Comment:
can you do this with a dll? you would get better performance on average, Python has to resize its storage for the list you're using so every now and then you're going to get O(n) performance to add an item whereas you are guaranteed O(1) performance on add with the dll. Since we're never accessing anything in the middle we don't need the index access that the Python list offers.

Reply:
Changed to a dll